src/hotspot/share/ci/ciInstance.cpp

```diff
@@ -42,16 +42,16 @@
 ciType* ciInstance::java_mirror_type() {
   VM_ENTRY_MARK;
   oop m = get_oop();
-  // Return NULL if it is not java.lang.Class.
-  if (m == NULL || m->klass() != vmClasses::Class_klass()) {
-    return NULL;
+  // Return null if it is not java.lang.Class.
+  if (m == nullptr || m->klass() != vmClasses::Class_klass()) {
+    return nullptr;
```

Comment:
I am confused. Did I miss some previous reviews which asked to keep `null` in comments?
These lines are confusing: comment says `null` but method returns `nullptr`. Why not use `nullptr` in comment?